migrations/V2__Add_Domains.sql

```diff
@@ -2,3 +2,5 @@ CREATE DOMAIN EUA_ID AS TEXT
 CONSTRAINT CHECK_VALID_EUA_ID CHECK (
     VALUE ~ '^[A-Z0-9]{4}$'
 );
+
+CREATE DOMAIN ZERO_STRING AS TEXT CHECK (length(value) > 0);
```

Comment:
To point it out,  right now, this checks for '',  but it doesn't check for ' ' or '  '. This is ok, but we aren't handling situations where users are just inserting spaces.

Reply:
I think this is fine, since the zero value for a string is `""` and handling that case alone I think is sufficient for now 👍